### DISCUSSION TITLE
Plans to use a more up to date rclone version?

### DISCUSSION THREAD

**Discussion by varzaman** [haridusministeerium/docker-seedbox-rclone-fetch-extract#4] [Ideas]
Hello,

I was just wondering if there were any plans on using a more current version of rclone. I see that the image is set up to use whatever is in Alpine:Linux's repo, which at this point is a year old(1.62 I believe).

I've been having some "corruption issues" recently where some files that rclone downloads aren't able to be read (even though it has the full file size). I've been using rclone 1.65, and haven't had any issues like that yet.

**Comment by laur89** [comment]
@varzaman bumped alpine; rclone `1.65.0-r3` is now packaged with the latest image.

**Reply by varzaman** [reply]
Hey just wanted to follow up. Ever since the update, there hasn't been a single problem with rclone. 👍
